pycbc/inference/models/tools.py

```diff
@@ -45,16 +45,16 @@ def draw_sample(loglr, size=None):
     cdf = numpy.exp(loglr).cumsum()
     cdf /= cdf[-1]
     xl = numpy.searchsorted(cdf, x)
+    
+    loglr = loglr - logsumexp(loglr)
```

Comment:
Thanks for the fix. What's the difference between normalization on `cdf` of `loglr` (previous code) and on `loglr` itself (this PR)? Can we remove the previous code? Here you generate `xl` twice.

Reply:
I'll need to remove this. I accidentally left it in for debugging purposes. I wasn't quite sure where the problem was originally. It's not here, it was the fact that the prior normalization was tied to the samples rather than upfront. This caused a large uncertainty depending on how the samples ended up distributed.